floors/fetcher.go

```diff
@@ -0,0 +1,294 @@
+package floors
+
+import (
+	"container/heap"
+	"context"
+	"encoding/json"
+	"errors"
+	"io"
+	"math"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/alitto/pond"
+	validator "github.com/asaskevich/govalidator"
+	"github.com/golang/glog"
+	"github.com/patrickmn/go-cache"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+var refetchCheckInterval = 300
```

Comment:
This is very specific to price floor implementation, after this interval, fetcher will check if there are any urls available to fetch. Its internal implementation value.
In order to configure this, it will require fetcher implementation knowledge, we can make it configurable
Let me know your thoughts where we can keep the configuration and what is the minimum and maximum value